src/scripts/slurm_status.py

```diff
@@ -358,6 +358,13 @@ def get_finished_job_stats(jobid, cluster):
                 log("Failed to parse CPU usage for job id %s: %s" % (jobid, row["SystemCPU"]))
                 raise
 
+        if row["AllocCPUS"] != "" and row["JobID"] == jobid:
+            try:
+                return_dict['CpusProvisioned'] = int(row["AllocCPUS"])
+            except:
```

Comment:
Normally i like to handle process termination separately from other exceptions,

```suggestion
            except SystemExit:
                raise
            except KeyboardInterrupt:
                raise
            except:
```

but if you think that's sufficiently irrelevant here (since it's not meant to be run interactively?) i guess it's fine as you have it too.